# Hypothesis Testing: Weekly Operating Cost Analysis

## Objective
To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

## Given Data
- Cost model: **W = 1000 + 5X**
- Number of restaurants in sample: **25**
- Sample mean weekly cost: **3050 Rs**
- X follows a normal distribution:
  - Mean (μₓ) = **600 units**
  - Standard deviation (σₓ) = **25 units**
- Significance level (α) = **0.05**

# Step 1: State the Hypotheses

Theoretical weekly cost using the model:
W = 1000 + 5X  
X (mean) = 600  
So, μ = 1000 + 5(600) = **4000 Rs**

### Null Hypothesis (H₀):
The mean weekly operating cost is **less than or equal to 4000 Rs**

### Alternative Hypothesis (H₁):
The mean weekly operating cost is **greater than 4000 Rs**

# Step 2: Calculate the Test Statistic

We use the Z-test formula:

\[
z = \frac{\bar{x} - \mu}{\sigma / \sqrt{n}}
\]

Where:
- \(\bar{x} = 3050\)  
- \(\mu = 4000\)  
- \(\sigma = 5 \times 25 = 125\)  
- \(n = 25\)

In [2]:
import math
from scipy.stats import norm

# Given data
sample_mean = 3050
mu = 1000 + 5 * 600   # Theoretical mean cost
sigma = 5 * 25        # Standard deviation
n = 25
alpha = 0.05

# Standard Error
SE = sigma / math.sqrt(n)

# Z-test statistic
z = (sample_mean - mu) / SE

# Critical value for a one-tailed test
critical_value = norm.ppf(1 - alpha)

z, critical_value

(-38.0, np.float64(1.6448536269514722))

# Step 3: Probability and Comparison

- Computed Z value: (printed above)
- Critical Z value at α = 0.05: **1.645**

Decision rule:
- If z > 1.645 → Reject H₀
- Otherwise → Fail to Reject H₀

In [3]:
# Decision

if z > critical_value:
    print("Reject H0: Costs have increased.")
else:
    print("Fail to Reject H0: No evidence that costs increased.")

Fail to Reject H0: No evidence that costs increased.


# Step 4: Conclusion

The calculated Z value is far below the critical value of 1.645.

### Final Conclusion:
We **fail to reject the null hypothesis (H₀)**.

There is **no significant statistical evidence** that weekly operating costs have increased beyond what the theoretical cost model suggests.

Therefore, the restaurant owners' claim of increased weekly operating costs is **not supported** by the sample data.